src/components/ui/Popover.tsx

```diff
@@ -0,0 +1,29 @@
+import { Popover as KPopover } from "@kobalte/core"
+import type { JSX } from "astro/jsx-runtime"
+import xIcon from "lucide-static/icons/x.svg?raw"
+
+type PopoverProps = {
+  trigger?: JSX.Element
+  content?: JSX.Element
+  title?: JSX.Element
+}
+
+export function Popover(props: PopoverProps) {
+  return (
+    <KPopover.Root>
+      <KPopover.Trigger class="transition-transform hover:scale-125 hover:opacity-80">{props.trigger}</KPopover.Trigger>
```

Comment:
The Popover is not triggered when the element is hovered so I added some transformations to make it clearer that you can click.